### ISSUE TITLE
Improve the color contrast on the home page links

### ISSUE THREAD

**Issue by kentcdodds** [javascriptair/site#118]
The color contrast here is really bad:

<img width="736" alt="screen shot 2016-06-15 at 2 54 47 pm" src="https://cloud.githubusercontent.com/assets/1500684/16097073/203b0634-3309-11e6-8559-1e2d3e2c0f3f.png">

The colors come from [this code](https://github.com/javascriptair/site/blob/601ea1e7276af2a02f67db5d5bb3e0d92382fb1a/src/pages/home/sections/episodes/episode-small.js#L77)

Thanks!

**Comment by kentcdodds** [comment]
Please see the [contribution guidelines](https://github.com/javascriptair/site/blob/master/CONTRIBUTING.md) and if this is your first PR to the site, please [add yourself to the contributors page](https://github.com/javascriptair/site/blob/master/CONTRIBUTING.md#add-yourself-to-the-contributors-page) :-)

**Comment by calvinclaus** [comment]
Will this be assigned or is this just first come first serve?

**Comment by kentcdodds** [comment]
First come first serve!

**Comment by kentcdodds** [comment]
I'd normally want to make something like this a [`first-timers-only`](http://kcd.im/first-timers-only) issue, and I will later if it's not resolved before I get around to it :-)

**Comment by calvinclaus** [comment]
I tried to keep it yellow to have it fit with the site. Would you be okay with this:
![bildschirmfoto 2016-06-15 um 23 13 55](https://cloud.githubusercontent.com/assets/6116771/16097641/146ec160-334f-11e6-9991-fdd667756589.png)

**Comment by kentcdodds** [comment]
Hmmm... I'm not sure I like that color... Would love to get the opinion of someone else on here. We should probably change the words `with` and `and` as well to just be a bit darker.
Also, could you check the colors with http://leaverou.github.io/contrast-ratio?

**Comment by calvinclaus** [comment]
The combination above would give us 4.3. 
Are you happy to move away from yellow completely?

**Comment by kentcdodds** [comment]
> The combination above would give us 4.3.

I think that's ok (no a11y expert here).

> Are you happy to move away from yellow completely?

Yeah, as long as it's not some other random color. Probably just a darker black I think... But still open to other suggestions :-)

**Comment by moaazsidat** [comment]
This tool by Adam Morse may be useful (and fun) for generating a cool a11y compliant color palette 😄  http://www.randoma11y.com

**Comment by calvinclaus** [comment]
> Yeah, as long as it's not some other random color.

Haha...

> Probably just a darker black I think... 

Oh. Well, if we don't care about differentiating between link and text color too much that's the easiest way...
![bildschirmfoto 2016-06-15 um 23 32 55](https://cloud.githubusercontent.com/assets/6116771/16098222/398a18fc-3352-11e6-902f-64f795cb1b46.png)
(8.2)

**Comment by kentcdodds** [comment]
Here are a few suggestions I got from this super awesome app [colorsafe.co](http://colorsafe.co/) ([thanks ahharissa](https://twitter.com/ahharissa/status/743193901918490624)!):

`#736012`

<img width="763" alt="screen shot 2016-06-15 at 3 38 36 pm" src="https://cloud.githubusercontent.com/assets/1500684/16098243/3fa34b7a-330f-11e6-9994-61ffaedadede.png">

`#555344`

<img width="759" alt="screen shot 2016-06-15 at 3 38 59 pm" src="https://cloud.githubusercontent.com/assets/1500684/16098249/4ccaedbc-330f-11e6-83ee-bfc44b27e39d.png">

`#634806`

<img width="757" alt="screen shot 2016-06-15 at 3 39 23 pm" src="https://cloud.githubusercontent.com/assets/1500684/16098257/5ac8fc1a-330f-11e6-87e4-d3e2de43c9fb.png">

Thoughts?

**Comment by kentcdodds** [comment]
I'm leaning toward that last one: `#634806`

**Comment by kentcdodds** [comment]
Thanks @markdalgleish, for bumping the font weight. I think it'd still be great to get a better color.

**Comment by rawrmonstar** [comment]
The sponsor links also fail WCAG 2.0 with a contrast ratio of 2.9.
![image](https://cloud.githubusercontent.com/assets/13814048/16099704/8df94262-330e-11e6-8d09-17fd3bf8d8fd.png)

If you replace it with the color you are leaning towards for the guest links (#634806), it passes with a contrast of 6.1:
![image](https://cloud.githubusercontent.com/assets/13814048/16099708/942b63ea-330e-11e6-8d61-1ddcd14da8d5.png)

**Comment by kentcdodds** [comment]
Thanks for noticing that @rawrmonstar! Would you like to make a pull request to fix that? You'll need to update [this](https://github.com/javascriptair/site/blob/1ec352e201e1e1e8e29abe4e849bdcc163909de7/src/pages/home/header/heading.js#L65) and [this](https://github.com/javascriptair/site/blob/1ec352e201e1e1e8e29abe4e849bdcc163909de7/src/pages/home/header/heading.js#L60).

Also, you could [add yourself as a contributor](https://github.com/javascriptair/site/blob/master/CONTRIBUTING.md#add-yourself-to-the-contributors-page) to get onto [the contributors page](https://javascriptair.com/contributors/) :-)
